pkg/apis/extensions/types.go

```diff
@@ -878,6 +878,14 @@ type AllowedHostPath struct {
 	// `/foo` would allow `/foo`, `/foo/` and `/foo/bar`
 	// `/foo` would not allow `/food` or `/etc/foo`
 	PathPrefix string
+
+	// ReadOnly when set to true will force volumes matching the path prefix to run as read-only.
```

Comment:
> will force volumes matching the path prefix to run as read-only

I don't think this should modify pod specs, which is what that sounds like.

Could be better phrased as "when set to true, will allow host volumes matching the pathPrefix if all volume mounts are readOnly", since you could have the following:

```
allowedHostPaths:[
{pathPrefix:"/foo", readOnly:true},
{pathPrefix:"/foo/bar", readOnly:false}
]
```

which would allow a writeable /foo/bar mount

Reply:
Ah, good catch. I copied the comment style from something does does modify the spec.